.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
                   command: echo $CIRCLE_JOB > circlejob.txt
 
             - restore_cache:
-                  key: deps1-{{ .Branch }}-{{ checksum "dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
+                  key: deps1-{{ .Branch }}-{{ checksum "dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
```

Comment:
This should be reverted right? As per @bpostlethwaite's [comment](https://github.com/plotly/dash-core-components/pull/373#issuecomment-437486409) and your [latest commit](https://github.com/plotly/dash-core-components/pull/373/commits/2e4b84fe03dc22cdaf8142d82c0d87cfda719bb1)

Reply:
Yeah, forgot that one, thanks!